webapp/channels/src/components/textbox/textbox.tsx

```diff
@@ -274,11 +274,11 @@ export default class Textbox extends React.PureComponent<Props> {
             textboxClassName += ' textarea--has-labels';
         }
 
-        if (this.props.preview) {
-            return (
+        return (
+            <>
                 <div
                     ref={this.wrapper}
-                    className='textarea-wrapper'
+                    className={classNames('textarea-wrapper', {'display-none': !this.props.preview})}
```

Comment:
Instead of adding a generic class called `display-none`, could you add a class named something like `textarea-wrapper-preview` and, based on that class, determine whether the `.textbox-preview-area` div or the `SuggestionBox` should have `display:none` set? I'm thinking something like
```scss
.textarea-wrapper.textarea-wrapper-preview {
    .textbox-preview-area {
        display: block;
    }
    .actual-textbox {
        display: none;
    }
}

.textarea-wrapper:not(.textarea-wrapper-preview) {
    .textbox-preview-area {
        display: none;
    }
    .actual-textbox {
        display: block;
    }
}
```

Reply:
awesome suggestion @hmhealey I have moved forward and used it 💯